README.md

```diff
@@ -112,7 +114,7 @@ Most users will want to work interactively with their Spark clusters. With the `
 ```bash
 aztk spark cluster ssh --id <my_cluster_id>
 ```
-By default, we port forward the Spark Web UI to *localhost:8080*, Spark Jobs UI to *localhost:4040*, and Jupyter to *localhost:8888*.
+By default, we port forward the Spark Web UI to *localhost:8080* and Spark Jobs UI to *localhost:4040*.
```

Comment:
change...

Reply:
what are we changing this to again?

Reply:
history server to localhost:18080 (if you have enabled the history server)

Reply:
right

Reply:
should be in your PR... but i'll add it anyway